### DISCUSSION TITLE
"Something went wrong with the SCSS compiler."

### DISCUSSION THREAD

**Discussion by richardpbull** [bootscore/bootscore#383] [Q&A] [answered]
Following this: https://bootscore.me/documentation/theme/bootstrap-css-sass/ I clearly made an error. Is there any way to easily include a line number where the error crashed the compiler?
Thank you.

**Comment by justinkruit** [comment, marked as answer]
Put the WordPress in development environment and you'll see more info.
```
define( 'WP_ENVIRONMENT_TYPE', 'development' );
```

**Reply by richardpbull** [reply]
Thank you

**Reply by justinkruit** [reply]
No problem! 😁
